crates/hir-expand/src/name.rs

```diff
@@ -79,7 +79,7 @@ impl Name {
     }
 
     /// Shortcut to create inline plain text name
-    const fn new_inline(text: &str) -> Name {
+    pub const fn new_inline(text: &str) -> Name {
```

Comment:
The constructors for this are deliberately private as to restrict the creation of names and from the looks of it (for this commit at least), this is not used in new places right?

(On that note I still want to rip out `SmolStr` of `Name`, we need a better symbol mechanism for the hir than this.)